xun/tests/test_transformations.py

```diff
@@ -354,3 +354,55 @@ def _xun_load_constants():
 
     ok, diff = check_ast_equals(generated, reference)
     assert ok, diff
+
+
+def test_load_symbolic_results_in_expression():
```

Comment:
I think it should be possible to write this test without source comparison 🤔 These tests are hard to update